routes/posts/comments/index.js

```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+const router = express.Router();
+import commentController from '../../../controllers/commentController.js';
+
+router.get('/', commentController.getComments);
```

Comment:
```suggestion
import { getComments } from '../../../controllers/commentController.js';

router.get('/', getComments);
```

요런식으로 쓰면 좀 더 깔끔할거 같아요..!

Reply:
me too.